lessons/python_basics/lesson.md

```diff
@@ -0,0 +1,57 @@
+# (Very, Very Brief) Introduction to Python
```

Comment:
Ah, ok, yea, that makes more sense. Looking forward to it!

On 15-07-09 03:31 PM, Charles Zhu wrote:
>
> In lessons/python_basics/lesson.md 
> <https://github.com/UofTCoders/studyGroup/pull/21#discussion_r34293738>:
>
> > @@ -0,0 +1,57 @@
> > +# (Very, Very Brief) Introduction to Python
>
> I definitely wasn't planning to go through the entire example file 
> (though I didn't realize I had overloaded the example to the point of 
> four hours). I was going to append the file afterwards to denote what 
> we did and didn't cover, and adding a few more links on topics we 
> didn't cover.
>
> I agree that something like "Overview of Python" would be more 
> appropriate a title.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/UofTCoders/studyGroup/pull/21/files#r34293738>.
>